x/rollup/keeper/deposits.go

```diff
@@ -22,21 +22,6 @@ import (
 	"github.com/samber/lo"
 )
 
-// setL1BlockInfo sets the L1 block info to the app state
-//
-// Persisted data conforms to optimism specs on L1 attributes:
-// https://github.com/ethereum-optimism/optimism/blob/develop/specs/deposits.md#l1-attributes-predeployed-contract
-func (k *Keeper) setL1BlockInfo(ctx sdk.Context, info types.L1BlockInfo) error { //nolint:gocritic
```

Comment:
Why is this moved to the other file and why is it exported?

Reply:
I decided to group it with the setter/getter for `Params` and the getter for `L1BlockInfo` directly inside of `keeper.go` to abstract away the store accessing logic in `deposits.go`. This follows the Cosmos SDK convention of avoiding direct store accessing in message/query handling logic.

I made it exported to allow it to be tested in `grpc_query_test.go`. If you don't think it (or any of the other store accessors) should be exported I can either remove the unit test file or include it in the `keeper` package instead of `keeper_test`. Lmk your thoughts on that.